### PR TITLE
Add exponential kernel to Graph

### PR DESCRIPTION
**Pull request by knaaptime**

### CODE REVIEW

**Review by martinfleis — APPROVED** [review]

File: libpysal/graph/_kernel.py [hunk 1]
```diff
@@ -106,6 +112,7 @@ def _kernel(
             - gaussian:
             - bisquare:
             - cosine:
+            - exponential:
```
Comment (knaaptime) [inline libpysal/graph/_kernel.py:115]: should these all probably have formulas here?

Comment (martinfleis) [reply libpysal/graph/_kernel.py:115]: Ideally

**Comment by knaaptime** [comment]
no way to bound it at the bandwidth if you want though 🤷‍♂️. would require another argument

**Comment by martinfleis** [comment]
Thinking about this more, I am actually not sure if I like passing alpha as `bandwidth`. It is inconsistent with the rest of the kernels then. Also, distance band has both bandwidth and alpha arguments (though not used together), so we shall aim for some degree of consistency between these.

**Pull request merged**